test/integrationtest/projecttest/graphql/schema/flamingo.me_flamingo-commerce_v3_cart_interfaces_graphql-Service.graphql

```diff
@@ -414,6 +422,7 @@ input Commerce_Cart_DeliveryShippingOption {
 extend type Query {
     Commerce_Cart: Commerce_DecoratedCart!
     Commerce_Cart_Validator: Commerce_Cart_ValidationResult!
+    Commerce_Cart_Validation_Restriction(marketplaceCode: String!, variantCode: String, deliveryCode: String!): Commerce_Cart_RestrictionResult!
```

Comment:
Not sure about the naming, by only having a look at the query name i couldn't figure out that we check product qty restrictions.. So maybe rename to `Commerce_Cart_Product_Restriction`?

Or at least add a comment so we have that extra information automatically in the GraphQL playground.

Reply:
Valid point..
I would not choose Product_Restriction because you may expect more than just qty..

I choose the name in analogy to the name in the domain layer (validation_restrictionService) - 
Maybe 
Commerce_Cart_QtyRestriction is better?

Reply:
Cool then lets go with `Commerce_Cart_QtyRestriction` :+1: